Composer/packages/server/src/server.ts

```diff
@@ -130,7 +130,7 @@ export async function start(): Promise<number | string> {
     // Dynamically search for an open PORT starting with PORT or 5000, so that
     // the app doesn't crash if the port is already being used.
     // (disabled in dev in order to avoid breaking the webpack dev server proxy)
-    port = await getPortPromise({ port: preferredPort as number });
+    port = await getPort({ port: preferredPort as number });
```

Comment:
@boydc2014  We did go away from using portfinder to avoid a race condition issue. GetPort holds on to a port started by a process for a 30 seconds to avoid port conflict. The preferred port is still the port 3000. Are u able to recreate the issue? Im not.